notes/stories/film/film-scifi-2020s.st.txt

```diff
@@ -889,3 +889,51 @@ anti-gravity technology [The HVEC flying vehicle was said to have anti-gravity e
 coping with the death of a spouse [The tragic death of Bernie Hayes' wife shaped him to become a raving conspiracy theorist.]
 
 
+movie: Dune (2021)
+==================
+
+:: Title
+Dune
+
+:: Description
+Dune (titled onscreen as Dune: Part One) is a 2021 American epic science
+fiction film directed by Denis Villeneuve and written by Villeneuve, Jon
+Spaihts, and Eric Roth. It is the first of a planned two-part adaptation of
+the 1965 novel by Frank Herbert, primarily covering the first half of the
+book.
+
+Synopsis: Set in the far future, it follows Paul Atreides as his family, the
+noble House Atreides, is thrust into a war for the deadly and inhospitable
+desert planet Arrakis.
+
+:: Date
+2021-09-03
+
+:: References
+https://en.wikipedia.org/wiki/Dune_(2021_film)
+
+:: Ratings
+2 <paul>
+
+:: Major Themes
+transnational conflict over a shared resource [Two rival aristocratic houses, the House of Atredies and the House of Harkonnen, battled for control over a vital resource, called "spice", which was found exclusively on the inhospitable desert planet Arrakis.]
+desert planet [Much of the story unfolded on inhospitable desert planet Arrakis.]
+imperialism [The House of Harkonnen, with the backing of the Emperor, was determined to gain control of Arrakis, crush its people, and gain control of the planet's deposits of "spice" - the most vital resource in the universe.]
+mother and son [Paul and his mother Lady Jessica were featured throughout.]
+father and son [Duke Leto was grooming his son, Paul, to succeed him as leader of the House of Atredies.]
+
+:: Minor Themes
+interstellar space travel [It was mentioned that the substance "spice" made navigating among the stars possible.]
+speculative spaceship [A fleet of Harkonnen vessels was shown over the planet Arrakis. A large, cigar-shaped vessel with a giant maw was shown over the planet Caladan.]
+force field generator [People surrounded themselves in personal force fields for defense. Most notably, Paul and Duncan had a practice knife fight while enveloped in such protective energy fields.]
+uncle and nephew [Glossu Rabba was the minion of his evil uncle, Baron Harkonnen.]
+coping with excruciating pain [Paul had to keep his hand inside of a pain box as part of a Bene Gesserit test.]
+clairvoyance [Paul inexplicably knew how to correctly don a "stillsuit" without ever having suited up in one before. The veteran Fremen ally Liet-Kynes was duly impressed with Paul as a result.]
+speculative environmental suit [The "stillsuits" recycled water and other waste products, enabling their wearers to survive in the desert for extended periods of time.]
+human familial relationship [Leto asked his consort, Lady Jessica, to protect their son, Paul, should anything happen to Leto.]
+mind control ability [Paul and Lady Jessica used a Bene Gesserit mind control technique on their captors.]
```

Comment:
Knock up? I think paul's abilities are central to the story, and my understanding is that he combines mind control with the ability to see into the future

Reply:
It strikes me that he is some kind of saviour figure, like Jesus, as well - not sure if we have a theme for that..

Reply:
Agree. Updated accordingly.